### PR TITLE
feat: Moved larger style components to different files

### LINKED ISSUE #387: Moving Stylus styles to a separate file
**Is your feature request related to a problem? Please describe.**
Currently, the Stylus styles are included in the VueJS file, which can make the code difficult to manage and maintain. 

**Describe the solution you'd like**
To improve code organization and readability, we should move the Stylus styles to a separate file. This can be achieved by creating a new file with a .styl extension and moving the Stylus styles to it. We can then link to this file in the HTML file using a <link> tag. This will improve the overall structure of our codebase and make it easier to maintain and update the styles.

The targetted files are:
- [Contact.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Contact.vue)
- [Corona.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Corona.vue)
- [Footer.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Footer.vue)
- [GlobalReport.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/GlobalReport.vue)
- [Header.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Header.vue)
- [Home.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Header.vue)
- [PageNotFound.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/PageNotFound.vue)
- [PersonalPage.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/user/PersonalPage.vue)

### PR DESCRIPTION
**Pull request by J0SAL**
Refactored styles for improved maintainability
## Description

This pull request refactors the styles for improved organization and maintainability. This change was made to reduce the clutter in the src/view files and make it easier to manage the styles.

Files Changed and corresponding stylus files,
- [Contact.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Contact.vue) -> src/styles/contact.styl
- [Corona.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Corona.vue) -> src/styles/corona.styl
- [Footer.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Footer.vue) -> src/styles/footer.styl
- [GlobalReport.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/GlobalReport.vue) -> src/styles/globalReport.styl
- [Header.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Header.vue) -> src/styles/header.styl
- [Home.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/Header.vue) -> src/styles/home.styl
- [PageNotFound.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/PageNotFound.vue) -> src/styles/pageNotFound.styl
- [PersonalPage.vue](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/user/PersonalPage.vue) -> src/styles/user/personalPage.styl

## Motivation and Context

fixes #387 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✅] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [✅] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by J0SAL** [comment]
@thisiskaransgit could you please review it

File: src/styles/globalReport.styl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  border-right 1px solid #F9F8F8
+  border-bottom 1px solid #F9F8F8
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+.IHR_charts-title
+  width 100%
+  margin-top 10px
+  border-top 1px solid #F9F8F8
+  border-left 1px solid #F9F8F8
+  border-right 1px solid #F9F8F8
+  border-bottom 1px solid #F9F8F8
```
Comment (thisiskaransgit) [inline src/styles/globalReport.styl:45]: this block can be easily optimized further

**Comment by thisiskaransgit** [comment]
> Hey @J0SAL, my approach would be to move the repeating elements to the global styles along with that you can extend the global style for all variables including the headlines, paragraphs, and hyperlinks as these are being used multiple times but their default value hasn't been set. We have to focus on reducing code redundancy too.
> 
> And you can let the styles unique to the components be as it is. As for the header and footer, you can create separate style files.
> Actually this is much needed, and along with that, there are custom elements created even though we use quasar variables.

You have only moved the Styles to a separate file, which does not fulfill the task, there are still multiple lines of unused CSS and code redundancy, keeping the unique variable aside, we should be moving the other variable to the global styles

File: src/styles/contact.styl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      @media screen and (max-width: 600px)
+        font-size 18pt
+
+    & > p
+      font-size 18pt
+      width 100%
+      margin 30pt auto
+      text-align left
+      @media screen and (max-width: 600px)
+        font-size 12pt
```
Comment (thisiskaransgit) [inline src/styles/contact.styl:52]: Rather than defining the variables here, we can move them to global and create multiple styles for specific use cases

**Comment by thisiskaransgit** [comment]
> Hey @J0SAL, my approach would be to move the repeating elements to the global styles along with that you can extend the global style for all variables including the headlines, paragraphs, and hyperlinks as these are being used multiple times but their default value hasn't been set. We have to focus on reducing code redundancy too.
> 
> And you can let the styles unique to the components be as it is. As for the header and footer, you can create separate style files.
> Actually this is much needed, and along with that, there are custom elements created even though we use quasar variables.
> 

You have only moved the Styles to a separate file, which does not fulfill the task, there are still multiple lines of unused CSS and code redundancy, keeping the unique variable aside, we should be moving the other variable to the global styles

File: src/styles/globalReport.styl (hunk 1 above)
Comment (J0SAL) [reply src/styles/globalReport.styl:45]: By optimization, do you mean creating a `border 1px 1px 1px 1px solid #F9F8F8` instead of 4 different lines?

**Comment by J0SAL** [comment]
> > Hey @J0SAL, my approach would be to move the repeating elements to the global styles along with that you can extend the global style for all variables including the headlines, paragraphs, and hyperlinks as these are being used multiple times but their default value hasn't been set. We have to focus on reducing code redundancy too.
> > And you can let the styles unique to the components be as it is. As for the header and footer, you can create separate style files.
> > Actually this is much needed, and along with that, there are custom elements created even though we use quasar variables.
> 
> You have only moved the Styles to a separate file, which does not fulfill the task, there are still multiple lines of unused CSS and code redundancy, keeping the unique variable aside, we should be moving the other variable to the global styles.
> 
> If you prefer to do it in parts, you can dissect the work in multiple PRs, which would be easy for you to work on and review with us.

I didn't get the dissecting PR part. Could you please explain how I can divide the work in multiple PRs and also what things do I need to do to close this PR?

**Comment by thisiskaransgit** [comment]
1. update global styles with stylus that are similar and import them to components
2. reduce the style for individual component 
I meant by creating separate PRs for kinds of commit.

**Comment by dpgiakatos** [comment]
The application has been upgraded to Vue 3.js with a new file structure. As this pull request has been inactive for an extended period, I am closing it.

**Pull request closed without merging**